meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -6,17 +6,17 @@ PREFERRED_VERSION_linux-yocto = "4.16.0%"
 PREFERRED_PROVIDER_virtual/bootloader ?=  "u-boot"
```

Comment:
Changing permissions above is a mistake. I will change to 664 as the other files are.